### PR TITLE
Home tab

### PR DESCRIPTION
**Pull request by Lrthayer**
I know the view is pretty lacking, but I wanted your guy's input on how we should display the user's info. At least we have the functionality in case we don't get to it.

### CODE REVIEW

File: app/controllers/DashboardController.js [hunk 1]
```diff
@@ -118,11 +117,39 @@ DashboardController.prototype.contacts = function () {
...
+        request
+         .get (blueprint.app.configs.apiserver.baseuri + route)
+         .set ('Authorization', 'bearer ' + token)
+         .end (function (err, resp) {
+             if (err) {
+                 if (err.status == '400') {
+                   console.log('we got a 400');
+                   //return done (null,false,{message: "Error Sending Message"});
+                 }
+                 else
```
Comment (DannyPeck) [inline app/controllers/DashboardController.js]: I think you're missing a bracket and having some weird whitespace issues ;)

**Review by DannyPeck — COMMENTED** [review]
So, I think it looks good, we need to figure out how to make it fit inside the window and not scroll sideways.  We also need to update the way we do our tables to use headers.  I think we are adding our headers as entries which is wrong.

File: public_html/dashboard.js [hunk 2]
```diff
@@ -9,6 +9,15 @@ $(document).ready(function () {
   $('#home').on('click', function () {
     $('.tab').hide();
     $('.table').empty();
+    
+    $.getJSON("/dashboard/userInfo", function (userInfo) {
+		$('#userName').html("username: " + userInfo.username);
```
Comment (DannyPeck) [inline public_html/dashboard.js]: Really nitpicky but should we Uppercase the strings on the left, like Username: ?

**Review by DannyPeck — APPROVED** [review]
Looks great!  Fantastic job!

File: public_html/dashboard.js (hunk 2 above)
Comment (Lrthayer) [reply public_html/dashboard.js]: yeah, I agree, I will change that.

**Pull request merged**